contributing-guides/maintainers-guide.md

```diff
@@ -56,22 +56,20 @@ as a guideline for current and future maintainers.
 
 - PRs should be merged once they
```

Comment:
```suggestion
- PRs should be merged once they pass all following points:
```

Reply:
I disagree.

Reply:
do you disagree with the content or the wording? It makes a previously implicit point explicit, reducing reader friction

Reply:
I disagree with the content. I think it interrupts the reader's flow a bit.